pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -160,18 +160,18 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 			oldDeserved := attr.deserved.Clone()
 			attr.deserved.Add(remaining.Clone().Multi(float64(attr.weight) / float64(totalWeight)))
 
-			if attr.capability != nil && !attr.deserved.LessEqual(attr.capability, api.Infinity) {
-				attr.deserved = helpers.Min(attr.deserved, attr.capability)
-				attr.deserved = helpers.Min(attr.deserved, attr.request)
-				meet[attr.queueID] = struct{}{}
-				klog.V(4).Infof("queue <%s> is meet cause of the capability", attr.name)
-			} else if attr.request.LessEqual(attr.deserved, api.Zero) {
-				attr.deserved = helpers.Min(attr.deserved, attr.request)
+			if attr.capability != nil {
```

Comment:
Sorry, which part of change fixes your problem? i'm not quite following you.

Reply:
refer to https://github.com/volcano-sh/volcano/issues/1850

Reply:
The calculated deserved value is wrong. 
In the process of calculating the deserved value, only some resources of deserved exceeds the capability and should not be stopped. Because other resources which smaller than capaliby may increase.

Reply:
Got it. I've noticed that `meet[attr.queueID] = struct{}{}` is removed when `!attr.deserved.LessEqual(attr.capability, api.Infinity)`.